packages/terra-clinical-header/src/Header.jsx

```diff
@@ -45,7 +45,7 @@ const defaultProps = {
 const Header = ({ children, title, startContent, endContent, isSubheader, ...customProps }) => {
   let startElement;
   if (startContent) {
-    startElement = <div className={cx('flex-end')}>{startContent}</div>;
+    startElement = <div className={cx('start-content flex-end')}>{startContent}</div>;
```

Comment:
the styling for this isn't being applied. To fix this, update how you are binding these class names.

Should be:
`className={cx(['start-content', 'flex-end'])}